src/borg/archiver.py

```diff
@@ -626,7 +626,7 @@ def chunkit(chunker_name, *args, **kwargs):
         print("KDFs (slow is GOOD, use argon2!) ===============================")
         count = 5
         for spec, func in [
-            ("pbkdf2", lambda: Passphrase('mypassphrase').kdf(b'salt'*8, PBKDF2_ITERATIONS, 32)),
+            ("pbkdf2", lambda: FlexiKey.pbkdf2(Passphrase('mypassphrase'), b'salt'*8, PBKDF2_ITERATIONS, 32)),
```

Comment:
As Passphrase is just a subclass of `str` with some additional functionality (not relevant here), you can just give 'mypassphrase' directly.